src/htmlgen.cpp

```diff
@@ -464,6 +464,69 @@ static QCString substituteHtmlKeywords(const QCString &file,
 
     switch (mathJaxVersion)
     {
+      case MATHJAX_VERSION_t::MathJax_4:
```

Comment:
Looks like a lot of code duplication between this and the MathJax_3 case. Maybe one or more helper functions could be used.

Reply:
Might be possible but when implementing support for Mathjax 4 it was unclear what exactly would be needed, hence some duplication.